src/libm-tester/testerutil.c

```diff
@@ -124,7 +124,11 @@ int readln(int fd, char *buf, int cnt) {
 }
 
 int startsWith(char *str, char *prefix) {
-  return strncmp(str, prefix, strlen(prefix)) == 0;
+  while(*str != '\0' && *prefix != '\0') {
```

Comment:
Are you sure this code is faster then the library call? I believe that strncmp have highly specialized implementations in glibc...

Reply:
The original code is strncmp + strlen.
This code is faster if defined as a static function.
I will move this to testerutil.h.